src/utils/metadata.ts

```diff
@@ -261,7 +261,20 @@ export function getInitialPublishFormDatasetsValues(
   return localStorageValues || initialValues
 }
 
-export function transformPublishFormToMetadata(
+const verifySelfDescription = async (args: {
+  url?: string
+  raw?: any
```

Comment:
Please rename this to sd or selfDescription. It becomes confusing, since raw contains the SD, whilst later a raw param is used as a boolean

Reply:
Fixed!